### DISCUSSION TITLE
Can I use systemd-genie with dnsmasq?

### DISCUSSION THREAD

**Discussion by trallnag** [arkane-systems/genie#202] [Q&A] [answered]
Thanks for the initiative of bringing systemd to WSL! For better or worse, systemd is in my opinion nowadays an integral part of Linux

**Comment by cerebrate** [comment, marked as answer]
Thanks muchly.

As for the question in the title, while I haven't tried it myself, I don't think there's any reason why it wouldn't work.

**Reply by trallnag** [reply]
Yes I got it to work in the mean time and it works great. I will set this question to solved once I add a short summary here for anyone who stumbles across it

**Reply by trallnag** [reply]
https://github.com/arkane-systems/genie/issues/187#issuecomment-922293304
